stytra/stimulation/stimuli/generic_stimuli.py

```diff
@@ -111,7 +135,7 @@ def stop(self):
         """
         pass
 
-    def initialise_external(self, experiment):
+    def initialise_external(self, experiment, environment_state: EnvironmentState = None):
```

Comment:
I don't think keeping both arguments is a good idea, at some point we anyway need to pass only the environment_state. We can consider having a `dev` branch to merge such PRs into, and then bump the version to 0.9 after mergining this. Though I don't think custom code relies much on modifying `intialise_external` (you can check for usage in `stytra_config`